Initial 1.3 support

Superfly-Johnson opened the pull request:
Did a large cleanup of irrelevant files and ported the framework to 1.3. There may be some bugs, i'm not extremely familiar with how Rimworld works as a platform but here you go.

dninemfive commented:
Thanks! Looks fine to me, but I'll have to check it on my end ofc. Will probably take a while lol

Superfly-Johnson commented:
My pleasure. 

I've simply removed deprecated functions and replaced them with equivalent ones for the record. 

Cleaned up the commits a bit, they should be easier to analyse now :)

Commit:
1.3 update
